pkg/crow/crow.go

```diff
@@ -145,7 +150,21 @@ func newCrow(cfg Config) (*Crow, error) {
 	apiCfg := api.Config{
 		Address: cfg.PrometheusAddr,
 	}
-	if cfg.UserID != "" {
+	if cfg.UserID != "" && cfg.PasswordFile != "" {
+		rt := &http.Transport{
+			Proxy: http.ProxyFromEnvironment,
+			DialContext: (&net.Dialer{
+				Timeout:   30 * time.Second,
+				KeepAlive: 30 * time.Second,
+			}).DialContext,
+			TLSHandshakeTimeout: 10 * time.Second,
+			TLSClientConfig: &tls.Config{
+				InsecureSkipVerify: true,
+			},
+		}
```

Comment:
I prefer to use the `DefaultRoundTripper` here but I was getting x509 certificate errors when hitting cortex endpoints (with valid certificates) so I've added this temporarily.  Is it possible we need to update `seego` to make sure we have latest CA certificates in the container?

Reply:
To simplify this for now, I could add another flag to toggle `InsecureSkipVerify` so this isn't the default 🤔